src/styles.css

```diff
@@ -399,13 +410,18 @@
   display: flex;
   flex-direction: column;
   justify-content: space-between;
+  text-decoration: none;
 }
 .cio-result-card-title {
   color: var(--gray-dust-500);
   margin: 0;
   margin-bottom: 1rem;
   font-weight: 700;
 }
+a .cio-result-card-title:hover {
+  color: var(--blue-denim-500);
+  text-decoration: underline;
```

Comment:
Is this a requirement? Is there anything else we can do to satisfy this? I don't love this
![quiz-a11y](https://github.com/Constructor-io/constructorio-ui-quizzes/assets/2954126/6b760435-fb4d-4a4f-b656-1f7f85504793)